src/DynamoCoreWpf/Utilities/ActionDebouncer.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Threading;
+using System.Windows.Threading;
+
+namespace Dynamo.Wpf.Utilities
+{
+    internal class ActionDebouncer
+    {
+        Dispatcher _dispatcher;
```

Comment:
style thing, we don't follow this naming pattern in this repo. (remove the underscores)